bin/configurer/add-roles.sh

```diff
@@ -11,15 +11,9 @@ ${dir}/utils/idam-add-role.sh "caseworker-publiclaw"
 # User used during the CCD import and ccd-role creation
 ${dir}/utils/idam-create-caseworker.sh "ccd.docker.default@hmcts.net" "ccd-import"
 
-${dir}/utils/idam-add-role.sh "caseworker-publiclaw-solicitor"
-${dir}/utils/ccd-add-role.sh "caseworker-publiclaw-solicitor"
-${dir}/utils/idam-add-role.sh "caseworker-publiclaw-courtadmin"
-${dir}/utils/ccd-add-role.sh "caseworker-publiclaw-courtadmin"
-${dir}/utils/idam-add-role.sh "caseworker-publiclaw-cafcass"
-${dir}/utils/ccd-add-role.sh "caseworker-publiclaw-cafcass"
-${dir}/utils/idam-add-role.sh "caseworker-publiclaw-gatekeeper"
-${dir}/utils/ccd-add-role.sh "caseworker-publiclaw-gatekeeper"
-${dir}/utils/idam-add-role.sh "caseworker-publiclaw-systemupdate"
-${dir}/utils/ccd-add-role.sh "caseworker-publiclaw-systemupdate"
-${dir}/utils/idam-add-role.sh "caseworker-publiclaw-judiciary"
-${dir}/utils/ccd-add-role.sh "caseworker-publiclaw-judiciary"
+roles=("-solicitor" "-courtadmin" "-cafcass" "-gatekeeper" "-systemupdate" "-judiciary")
```

Comment:
For me it would look better if role separating dash was removed from here and kept in lines 17-18.

Reply:
good point, changed